Sources/Node/Utilities/Errors.swift

```diff
@@ -12,6 +12,6 @@ extension NodeError {
 
 extension NodeError: CustomStringConvertible {
     public var description: String {
-        return "Expected \(expectation)\nGot: \(node)\nKey: \(key)"
+        return "Expected \(expectation)\nGot: \(node ?? .null)\nKey: \(key ?? [])"
```

Comment:
`\n` isn't ideal in JSON error messages.

Something like this is better:

```swift
extension NodeError: CustomStringConvertible {
    public var description: String {
        let nodeDescription: String

        if let node = node {
            nodeDescription = "\(node)"
        } else {
            nodeDescription = "nil"
        }

        let keyDescription: String
        if let key = key {
            keyDescription = key.description
        } else {
            keyDescription = "''"
        }

        return "Expected \(expectation) got \(nodeDescription) for key \(keyDescription)"
    }
}
```

https://github.com/vapor/node/issues/39